packages/terra-search-field/src/SearchField.jsx

```diff
@@ -183,36 +213,64 @@ class SearchField extends React.Component {
       inputAttributes,
       ...customProps
     } = this.props;
+
+    const clearButtonVisible = !this.state.searchText ? 'input-clear' : undefined;
+
+    const clearButtonClassNames = cx([
+      'clear',
+    ]);
```

Comment:
Since there is only one class name here, let's just use `className={cx('clear')}` below

Reply:
Updated: [6421e1b](https://github.com/cerner/terra-core/pull/2317/commits/6421e1b29105ddc7726dccabaf5e49a2238a34df)